### ISSUE TITLE
Why anaconda cannot autocomplete some third-party modules in python3

### ISSUE THREAD

**Issue by zhb1207** [DamnWidget/anaconda#728]
## Expected Behaviour
Hi I installed some packages such as PIL, seleniums and BeautifulSoup.
And when I tried to import these packages use 
"from   import  " it doesn't work well.
Here are the results:
![image](https://user-images.githubusercontent.com/15380695/33468898-7339f97c-d69a-11e7-996e-42cc17b7b754.png)
![image](https://user-images.githubusercontent.com/15380695/33468925-b5d3c650-d69a-11e7-9c2b-76c5d9da4636.png)
![image](https://user-images.githubusercontent.com/15380695/33468934-c360b21a-d69a-11e7-8e76-bb798eda1d6a.png)

I am wondering that why some of them is working well some is not. : )
Thanks.

**Comment by DamnWidget** [comment]
This can happen for two different things
1) Because your configured python interpreter/environment for anaconda differs from the one were those libraries are installed, and Jedi can't see them
2) Because anaconda's JsonServer crashed and anaconda is not really working 

That is why we ask for logs versions and other information that you are not offering in your report. We need more information in order to help you.

**Comment by zhb1207** [comment]
Thanks for your reply : )
May I ask when can I find the logs or versions information related to this problem?

**Comment by zhb1207** [comment]
Thanks for your suggestion.
I changed the interpreter in "Anaconda default settings" from "python_interpreter": python" to "python_interpreter": "C:/python3/python3.exe".
And put 
{
"python_interpreter": "C:/python3/python3.exe",
"suppress_word_completions":true,
"suppress_explicit_completions":true,
"complete_parameters":true,
}
in Anaconda user settings.
This time it works. 
However I remember that when I first installed Anaconda in st3, these settings were made once. Is there any action that would trigger this back to default?
